Makefile

```
@@ -54,3 +62,19 @@ generate-documentation:
 .PHONY: local-plugin-definitions
 local-plugin-definitions: kustomize yq
 	hack/local-plugin-definitions
+
+PLUGIN ?= "None"
+.PHONY: generate-readme
+generate-readme: helm-docs
+	if [ $(PLUGIN) == "None" ]; then\
+		echo "Please specify target plugin. make generate-readme PLUGIN=<plugin_name>"; \
+		exit 1;\
+	fi;\
+
+	if [ -d "./$(PLUGIN)" ]; then\
+		cd ./$(PLUGIN); ../bin/helm-docs -o ../README.md -t ./README.md.gotmpl;\
+	else \
+		echo "Error: The specified plugin directory '$(PLUGIN)' does not exist in this repository. Please check the plugin name and try again."; \
+		exit 1; \
+	fi
```

With the below suggestion, helm-docs will output to /dev/null 2>&1 to suppress unnecessary messages.
```suggestion
PLUGIN ?= "None"
.PHONY: generate-readme
generate-readme: helm-docs
	@PLUGIN=$(PLUGIN); \
	if [ "$$PLUGIN" == "None" ]; then \
		echo "Please specify target plugin. make generate-readme PLUGIN=<plugin_name>"; \
		exit 1; \
	fi; \
	if [ -d "./$$PLUGIN" ]; then \
		cd ./$$PLUGIN; ../bin/helm-docs -o ../README.md -t ./README.md.gotmpl > /dev/null 2>&1; \
	else \
		echo "Error: The specified plugin directory '$$PLUGIN' does not exist in this repository. Please check the plugin name and try again."; \
		exit 1; \
	fi
```

---

any reason to make this less verbose ? I was actually looking to add couple more echo to the helm-docs target

---

Because I don't want to see all the "if" statement information. I would like to mainly see only the output from `make-docs` tool + added information. 

For example see the below output. I don't want to see anything before "INFO[2025-02-26T10:00:49+01:00] Found Chart directories [charts]"

```shell
─❯❯❯ make generate-readme PLUGIN=alerts
/Users/I513945/development/greenhouse/greenhouse-extensions/bin/helm-docs is not expected. Removing it before installing.
GOBIN=/Users/I513945/development/greenhouse/greenhouse-extensions/bin go install github.com/norwoodj/helm-docs/cmd/helm-docs@latest
if [ alerts == "None" ]; then\
                echo "Please specify target plugin. make generate-readme PLUGIN=<plugin_name>"; \
                exit 1;\
        fi;\

if [ -d "./alerts" ]; then\
                cd ./alerts; ../bin/helm-docs -o ../README.md -t ./README.md.gotmpl;\
        else \
                echo "Error: The specified plugin directory 'alerts' does not exist in this repository. Please check the plugin name and try again."; \
                exit 1; \
        fi
INFO[2025-02-26T10:00:49+01:00] Found Chart directories [charts]             
INFO[2025-02-26T10:00:49+01:00] Generating README Documentation for chart charts 
```

---

agreed, but with `> /dev/null 2>&1;` on helm-docs we loose the INFO logs. regarding what is above that, looks like re-assigning the var helps but i'm looking further on why is that.

will also add more output in case of failure